domain/application/modelmigration/import_test.go

```diff
@@ -56,12 +56,13 @@ var _ = gc.Suite(&importSuite{})
 func (s *importSuite) TestApplicationImportWithMinimalCharm(c *gc.C) {
 	model := description.NewModel(description.ModelArgs{})
 
+	since := time.Now().UTC()
```

Comment:
since matches StatusInfo
```
// StatusInfo holds a Status and associated information.
type StatusInfo struct {
	Status  Status
	Message string
	Data    map[string]interface{}
	Since   *time.Time
}
```

Reply:
We use `since` in lots of other places, but renamed here.